spec/api-browser-window-spec.js

```diff
@@ -1923,14 +1922,23 @@ describe('BrowserWindow module', () => {
 
       it('validates process APIs access in sandboxed renderer', (done) => {
         ipcMain.once('answer', function (event, test) {
-          assert.strictEqual(test.arch, remote.process.arch)
-          assert.strictEqual(test.platform, remote.process.platform)
-          assert.deepStrictEqual(...resolveGetters(test.env, remote.process.env))
-          assert.strictEqual(test.execPath, remote.process.helperExecPath)
-          assert.strictEqual(test.sandboxed, true)
-          assert.strictEqual(test.type, 'renderer')
-          assert.strictEqual(test.version, remote.process.version)
-          assert.deepStrictEqual(test.versions, remote.process.versions)
+          expect(test.crash).to.equal('function')
```

Comment:
Looks like this is not exposed on linux?

Reply:
should be, there is no ifdef in the native code. will try to figure out what's wrong

Reply:
`process.getSystemMemoryInfo()` was throwing on Linux, which caused the whole `window.test` to be unavailable.